src/pages/CreatorCenter/components/ClassRegist/components/InputForm.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import ClassInfo from './ClassBasicInfo';
+
+function InputForm({
+  type,
+  title,
+  isMultiple,
+  subject,
+  description,
+  borderUnUse,
+  classInfo,
+  setClassInfo,
+  recordingFlag,
+  currentClass,
+  detailFlag,
+}) {
+  const classInfoSetter = event => {
+    classInfo[title] = event.target.value;
+    setClassInfo({ ...classInfo });
+  };
+
+  const INPUTS = {
+    list: (
+      <Select
+        onChange={classInfoSetter}
+        name="levelList"
+        defaultValue={
+          detailFlag === true && currentClass
+            ? currentClass[title]
+            : ClassInfo[title]
+        }
+      >
+        <option value="">선택안함</option>
+        <option
+          value="입문자"
+          selected={
+            detailFlag === true && currentClass
```

Comment:
```suggestion
            detailFlag  && currentClass
```
detailFlag가 이미 boolean 값이라면 위와 같이 수정할 수 있겠습니다.

아래 있는 코드들도 다 위와같이 수정해주세요!

Reply:
완료